common-arcana.lic

```diff
@@ -456,7 +458,10 @@ module DRCA
     command = data['prep'] if data['prep']
     command = data['prep_type'] if data['prep_type']
 
-    return unless prepare?(data['abbrev'], data['mana'], data['symbiosis'], command)
+    tm_tattoo = false
```

Comment:
💅 Minor inconsistency: Here the variable is named tm_tattoo but in the prepare method is spelled tattoo_tm

Reply:
I'm in favor of consistency

Reply:
Yeah, I renamed it a couple times, thank you.